lib/_http_server.js

```diff
@@ -348,8 +354,12 @@ function connectionListenerInternal(server, socket) {
     socket.setTimeout(server.timeout);
   socket.on('timeout', socketOnTimeout);
 
-  const parser = parsers.alloc();
-  parser.reinitialize(HTTPParser.REQUEST, parser[is_reused_symbol]);
+  var parser = parsers.alloc();
+
+  parser.initialize(
```

Comment:
```suggestion
  // TODO(addaleax): This doesn’t play well with the
  // `async_hooks.currentResource()` proposal, see
  // https://github.com/nodejs/node/pull/21313
  parser.initialize(
```

Reply:
Why you say so?

Reply:
@mcollina Because we don’t store the async resource in a programmatically accesible place, and we haven’t figured out how do to that yet.

Reply:
Awesome ok!